src/components/ButtonFollow.vue

```diff
@@ -32,6 +32,7 @@ const canFollow = computed(() =>
         'flex items-center justify-center hover:!border-red hover:!bg-red hover:!bg-opacity-5 hover:!text-red':
           isFollowing
       }"
+      :primary="!isFollowing"
```

Comment:
Is it a good idea UX wise to have the same button with the same meaning have 2 differents UI ?

Reply:
I don't see the issue. Also this idea comes from designers